pkg/api/keypair/formatter.go

```diff
@@ -29,6 +29,7 @@ func CollectionFormatter(request *types.APIRequest, collection *types.GenericCol
 type KeyGenActionHandler struct {
 	KeyPairs     v1alpha1.KeyPairClient
 	KeyPairCache v1alpha1.KeyPairCache
+	Config       config.Options
```

Comment:
We are passing the config everywhere mostly for the namespace. Is it better to store it as a setting?

Reply:
updated to passing namespace only where it is appliable.